CHANGELOG.md

```diff
@@ -16,6 +16,70 @@
 
 ---
 
+## v3.0.0
+
+Here is the list of all the breaking changes between version 2 and 3. For each change, we show the new way to handle your use case.
+
+### Network errors no longer monitored
+
+RUM browser SDK no longer automatically monitors network errors.
+
+**New option**: You can use the option beforeSend to get the resource, check the `status` and send it with the `addError()` API.
+
+### Source attribute of addError() has been removed
+
+You can no longer change the source of error sent with `addError()`. All these errors have a `custom` source.
+
+**New option**: If you used this feature to identify `network` and `source` errors, you can add context attributes with `addError()` instead.
+
+### Non versioned RUM CDN files no longer updated
+
+All these files are frozen on version 2.18.0:
+
+- www.datadoghq-browser-agent.com/datadog-rum.js
+- www.datadoghq-browser-agent.com/datadog-rum-recorder.js
+- www.datadoghq-browser-agent.com/datadog-rum-eu.js
+- www.datadoghq-browser-agent.com/datadog-rum-us.js
+
+**New option**: In order to get the last update of the sdk, you have to use versioned CDN files:
+
+| Old files                                            | New files              |
+| ---------------------------------------------------- | ---------------------- |
+| datadog-rum.js, datadog-rum-eu.js, datadog-rum-us.js | datadog-rum-slim-v3.js |
+| datadog-rum-recorder.js                              | datadog-rum-v3.js      |
```

Comment:
I have mixed feelings about this section.
IMO, it is not really a breaking change. If we document that, we should also document what is the difference between rum and rum-slim.
We could either leave that to the future official doc or add another section outside of breaking changes to explain those new artifacts.
wdyt?

Reply:
I agree. IMO, the goal of the section is mainly to warn NPM users. If the official doc doesn't take too long, we can wait. 
At worst, if we see confusion amongst CDN user, we can add a new section later on. For now, I'm gonna delete it. Does it sound good?

Reply:
👍